src/validation.js

```diff
@@ -209,7 +200,11 @@ export default class AuroFormValidation {
    * @returns {void}
    */
   getAuroInputs(elem) {
-    this.auroInputElements = elem.renderRoot.querySelectorAll('auro-input');
+    if (elem.hasAttribute('auro-input')) {
```

Comment:
This should be updated to use the new library method to see if `elem` is an auro-input.

```js
this.runtimeUtils.elementMatch('auro-input')
```

Reply:
Also, what is the use case for this wrapper?

Reply:
Basically I am trying to find a way to gather all elements that have the tag name or attribute `auro-input`. My logic here is wrong, so I will figure out a way to correct it.